### PR TITLE
adds basic scroll depth info and other updates

### PR DESCRIPTION
**Pull request by lizzerdrix**
* adds documentation around manual scroll depth implementation and automatic URL population
* removes iframe: true

### CODE REVIEW

**Comment by lizzerdrix** [comment]
@h55nick @andremalan @pnts whatcha think?

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+ruby-2.1.0
```
Comment (h55nick) [inline .ruby-version]: :eyes:

File: source/implementation.html.erb [hunk 2]
```diff
@@ -93,7 +92,16 @@ pid: '4fc608fea782f333ea0000fb'
...
+      <p><strong>url:</strong> <em class="required">(required)</em> We support automatic URL detection if your site headers contain <a href="https://developers.facebook.com/docs/sharing/best-practices#tags">Facebook Open Graph tags</a> (specifically 'og:url') or canonical URL meta tags in this format:
+      </p>
+      <% code("html") do %>
+<link rel="canonical" href="http://www.website.com/categories/article1.html" />
+      <% end %>
+      <p>
+      If your site contains either of these tags in its header, you do not need to include the 'url' field - it will populate automatically.
+      </p>
+      <p>   
+        <strong>If your site headers do not contain these tags, you will need to include the canonical URL in this field.</strong> This is the canonical or permanent URL of the content. This is the URL that a user would enter in a browser to visit the page directly. This should not include the protocol unless this changes the content being rendered. Trailing slashes should also be consistent.</p>
```
Comment (h55nick) [inline source/implementation.html.erb]: this is a great addition! if I were doing implementations I'd be so pumped about this. and even though I'm not, still sweet as.

File: source/implementation.html.erb [hunk 3]
```diff
@@ -158,14 +166,6 @@ tags: ['content analytics', 'content marketing', 'publishers']
...
     </li>
     
-    <li>
-      <p><strong>iframe:</strong>  <em class="recommended">(recommended)</em> This will drop an iframe on the page to ensure faster and non-blocking page loads. The default value is <b>true</b> and should <em>not</em> be changed.</p>
-    
-<% code("html") do %>
-iframe: true
-<% end %>
-    </li>
-
```
Comment (h55nick) [inline source/implementation.html.erb]: this will come back to haunt us. just not sure when.

File: source/scroll-depth.html.erb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    <p>
+    Whenever the user scrolls to a given quartile (i.e. percentage of content depth - 25, 50, 75, 100), create an event that runs the following code: 
+    </p>
+    
+    <% code("javascript") do %>
+    SPR.scrollDepthReached(percentage);
+    <% end %>
+
+    <p>
+    The only values accepted for percentage are 25, 50, 75, or 100.
```
Comment (h55nick) [inline source/scroll-depth.html.erb]: does it throw an error (or log one) if one is sent that isn't in here?

Comment (lizzerdrix) [inline source/scroll-depth.html.erb]: Yes! I can add that in.

File: .ruby-version (hunk 1 above)
Comment (andremalan) [inline .ruby-version]: :lollipop:

File: source/implementation.html.erb (hunk 2 above)
Comment (andremalan) [inline source/implementation.html.erb]: see if you can find a link to why using canonical urls (not the facebook one) is good

**Comment by andremalan** [comment]
Looks good @lizzerdrix. Remember to remove iframe: true from the example reach config in the basic implementation section.

File: .ruby-version (hunk 1 above)
Comment (lizzerdrix) [inline .ruby-version]: eyeballs and lollipop?

**Comment by lizzerdrix** [comment]
@andremalan @h55nick I believe I addressed everything - not totally sure where eyeballs vs. lollipop stands but otherwise are things good?

**Comment by h55nick** [comment]
:+1: think @pnts would want to take a look at this.

**Comment by pnts** [comment]
Taking a look now!

**Comment by pnts** [comment]
:+1: 
simba

**Pull request merged**